Source/Glass.Mapper.Sc/SitecoreVersionAbstractions.cs

```diff
@@ -35,6 +35,13 @@ public static string GetMediaUrl(MediaItem media)
 
         public static string GetItemUrl(Item item, UrlOptions urlOptions)
         {
+            //Added check for when rendering paramater request comes as Sitecore.Links.UrlBuilders.Helpers.ItemPathBuilder.GetRelativePath

```

Comment:
Seems like a better backward compatible fix is to copy out the previous null check based on the notes from #463